modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -1247,14 +1364,23 @@ struct InferList2: public cv::detail::KernelTag {
 
         const auto &input_name_0 = uu.params.input_names.front();
         const auto &mm_0 = in_metas[0u];
-        const auto &matdesc = cv::util::get<cv::GMatDesc>(mm_0);
+
+        if (!(cv::util::holds_alternative<cv::GMatDesc>(mm_0) ||
```

Comment:
Looks we don't need these checks once `isImage` handles everything inside

Reply:
Fixed, thanks! https://github.com/opencv/opencv/pull/24938/commits/833bd890ef003a7c161f3a91c61b9c8d81df0e55